modules/gitops-argo-cd-installation.adoc

```diff
@@ -23,4 +23,16 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 .. Create an external OS Route to access Argo CD server. Click *Server* -> *Route* and check *Enabled*.  
 
-. To open the Argo CD web UI, click the route by navigating to *Networking* -> *Routes* -> *<instance name>-server* in the project where the Argo CD instance is installed.
\ No newline at end of file
+. To open the Argo CD web UI, click the route by navigating to *Networking* -> *Routes* -> *<instance name>-server* in the project where the Argo CD instance is installed.
```

Comment:
We can maybe add the step like this?:
```suggestion
. Go to *Networking* -> *Routes* -> *<instance_name>-server* in the project where the Argo CD instance is installed.

. On the *Details* tab, click the Argo CD web UI link under *Route details* -> *Location*. The Argo CD web UI opens in a separate browser window.
```

Reply:
Fixed.

Reply:
Hello @Dhruv-Soni11 It might have been easy to overlook, but I wanted to address the missing step I highlight in the last two comments:
```
. On the *Details* tab, click the Argo CD web UI link under *Route details* -> *Location*. The Argo CD web UI opens in a separate browser window.
```
Thank you!